ironfish/src/assets/assetsVerificationApi.ts

```diff
@@ -5,8 +5,17 @@ import axios, { AxiosAdapter, AxiosError, AxiosRequestConfig, AxiosResponse } fr
 import url, { URL } from 'url'
 import { FileSystem } from '../fileSystems'
 
-type GetVerifiedAssetsResponse = {
-  assets: Array<{ identifier: string }>
+type AssetData = {
+  assetId: string
+  name: string
+  symbol: string
+  decimals: number
+  logoURI: string
+}
```

Comment:
While we're at this, it might be worth adding other fields, in particular: an URI/URL to point at a website describing asset. I'm not sure if this has been requested yet, but I certainly see some use cases for this